practice-app/website/api/auth.py

```diff
@@ -48,11 +48,7 @@ def login():
     if not user or not check_password_hash(user.password, password):
         return {'error': 'Incorrect email or password.'}, 401
 
-    access_token = create_access_token(identity=user)
-    return jsonify(access_token=access_token)
+    is_artist = Artist.query.get(user.id)
```

Comment:
Is the `is_artist` variable supposted to reference an Artist object? If so, naming it to just `artist` would be better. Everything else seems ok. Thank you for your efforts.

Reply:
It references an Artist object as you said. I used the variable to check whether an Artist object exists with the given id so I had named the variable `is_artist`. I can change it to `artist.`